enhancements/network/allow-mtu-and-overlay-port-changes.md

```diff
@@ -0,0 +1,185 @@
+---
+title: allow-mtu-and-overlay-port-changes
+authors:
+  - "@juanluisvaladas"
+reviewers:
+  - "@danwinship"
+  - "@mccv1r0"
+approvers:
+  - TBD
+creation-date: 2021-01-25
+last-updated: 2022-02-03
+status: provisional
+---
+
+# Allow MTU and overlay port changes
+
+This covers adding the capability to the cluster network operator of changing
+the MTU, and the VXLAN and Geneve ports post installation.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Customers may need to change the MTU, or the ports used for VXLAN or Geneve
```

Comment:
I'd suggest "Cluster administrators" instead of "Customers" to be a bit more direct about the target persona